source/concepts/overview.rst

```diff
@@ -0,0 +1,40 @@
+########
+Overview
+########
+
+Connect and combine Symbol's built-in features to architect your blockchain solution.
+Use our ready-made plugin configurations for your most common business needs, or configure the plugin primitives plugin to customize the capabilities of your network without sacrificing performance.
+
+*****************
+Transfer of value
+*****************
+
+* **Transfer Transaction**: The most basic transaction used to send mosaics and messages between two accounts. Read :doc:`more <transfer-transaction>`.
+* **Aggregate Transaction**: A type of transaction that merges multiple transactions into one by generating a one-time disposable smart contract. Read :doc:`more <aggregate-transaction>`.
+* **Cross-Chain Swap**: A built-in feature of Symbol which enables the trading of tokens across different blockchains without using an intermediary party (e.g. an exchange service). Read :doc:`more <cross-chain-swaps>`.
+
+*******************
+Accounts management
+*******************
+
+* **Account**: An account is a container for assets, which only you can modify with your private key.  Read :doc:`more <account>`.
```

Comment:
An account is a container for assets, which can only be modified with its private key.  Read :doc:`more <account>`.